core/master_derivable_key.go

```diff
@@ -88,6 +88,6 @@ func (master *MasterDerivableKey) Derive(relativePath string) (*HDKey, error) {
 }
 
 func validateRelativePath(relativePath string) bool {
-	match, _ := regexp.MatchString(`^(\/(\d\d?\d?))+$`, relativePath)
+	match, _ := regexp.MatchString(`^\/\d+(\/0)(\/0)?$`, relativePath)
```

Comment:
Why did this change?

Reply:
reverted to support more then 3 digits derivation  paths